tests/e2e/tests/istioctl/istioctl_test.go

```diff
@@ -0,0 +1,355 @@
+// Copyright 2017 Istio Authors
```

Comment:
This file has some long lines in it. At a minimum can you wrap the comments and maybe the printf/errorf function lines so its easier to read with side-by-side diff?